### ISSUE TITLE
Comment out Nar'Sie's endgame and just give Cultists immediate victory until the lag is fixed

### ISSUE THREAD

**Issue by dylanstrategie** [d3athrow/vgstation13#4455]
Cultist rounds are usually some of the worst rounds in the game for non-cultists, because no matter what you do, even if you literally kill everyone on sight, some random Cultist will slip away and summon Nar'Sie with 8 manifests in some closet off-station

And thus, we get to Nar'Sie's endgame. It's an horrible snorefest, lags down the server to a crawl for almost one minute (when we have one of the most robust server infrastructures in the entire SS13 community), then continues to freeze the server while dumping hostile mobs on everyone and letting them get 30 free hits before one can even pull off any action

I'm fully aware that Nar'Sie's arrivals marks the official end of the round, but after an already terrible round of barely watered powergaming only to see the Cult succeed for the hundredth time, I'd at least expect things to be over quickly. The lag drags on the already painful 15 minutes of waiting for the shuttle to dock and leave to an excruciating half an hour of feature films and powerpoint presentations

If you're not going to actually fix it, at least have the gal to comment it out. Given that Nar'Sie's arrivals means the Cult officially won, just make the Cult win when Nar'Sie is summoned, and cut out the Endgame altogether until it's fixed. Or switch back to old Nar'Sie, it's basically a Singularity, and the Singularity never lags that much

TL;DR Cult rounds are already boring and uninteresting snorefests that lead to a straight Cult win 99 % of the time unless you lolstubbedmytoe your way out of here. We don't need an equally laggy Endgame that is enjoyable to everyone and delays round end for no reason. Fix it or scrap it.

**Comment by d3athrow** [comment]
How do you suggest fixing the lag outside of removing narsie

**Comment by IratePirate** [comment]
SAVE NAR-SIE

**Comment by dylanstrategie** [comment]
- Actually checking and fixing what is wrong with the Endgame code, but given that it is N3X15Code(TM), this is going to be painful
- Switching back to old Nar'Sie behaviours, because we don't need the new Nar'Sie if it doesn't even work properly. Add victory when Nar'Sie is summoned and we're good to go
- Cutting shit out until we find what is wrong. Between every single tile on station being constantly rolled for degradation and power, atmos and lighting being mass-updated, there's only features to be had

**Comment by ComicIronic** [comment]
>some random Cultist will slip away and summon Nar'Sie with 8 manifests in some closet off-station

I thought this stopped being possible with the new balance changes.

>Switching back to old Nar'Sie behaviours, because we don't need the new Nar'Sie if it doesn't even work properly

Rollback is a bad idea unless you can find evidence in the system that it's necessary.

**Comment by AutismSpider** [comment]
new balance changes did nothing to stop that.

just eat ton of iron pills and some cheap food like bust nuts or pizza and that deal with the bleed out.
